staging/src/k8s.io/apiextensions-apiserver/test/integration/table_test.go

```diff
@@ -149,7 +149,9 @@ func TestTableGet(t *testing.T) {
 		parameterCodec := runtime.NewParameterCodec(scheme)
 		metav1.AddToGroupVersion(scheme, gv)
 		scheme.AddKnownTypes(gv, &metav1beta1.Table{}, &metav1beta1.TableOptions{})
+		scheme.AddKnownTypes(gv, &metav1.Table{}, &metav1.TableOptions{})
```

Comment:
I'm surprised these are getting added under the CRD's groupVersion... that doesn't seem correct at all. shouldn't these be added as `meta.k8s.io/v1[beta1]`?

Reply:
it's for the dynamic client, which only has one scheme.  without it, getting back a table breaks.

Reply:
Oh wait, you mean the GV of the CRD, yes probably shouldn't.  Let me test without it (it may have been the previous iteration of the config).

Reply:
Removed